### PR TITLE
Updated tabs topic

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/tabs.md [hunk 1]
```diff
@@ -26,21 +26,27 @@ The Tabs can be **fixed** and can fill up the available horizontal space by adap
...
 
-For the majority of scenarios, the Tabs need to contain between two and four tabs. If your case requires more than that, you may want to consider a fluid mode and represent only the tabs in view.
+For the majority of scenarios, the Tabs need to contain between two and four tabs. You can exclude a tab by setting it to ~No Symbol. If your case requires more tabs, you may want to consider a fluid mode and represent only the tabs in view.
 
 <img class="responsive-img" src="../images/tabs_2.png" srcset="../images/tabs_2@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_3.png" srcset="../images/tabs_3@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_4.png" srcset="../images/tabs_4@2x.png 2x" />
 
-## Type
+## Short Bar Tabs Content
```
Comment (StefanIvanov) [inline en/components/tabs.md:37]:
```suggestion
## Short Bar Content
```

File: en/components/tabs.md [hunk 2]
```diff
@@ -26,21 +26,27 @@ The Tabs can be **fixed** and can fill up the available horizontal space by adap
...
 
-The short Tabs support **text** and icon content modes to decribe the item in each tab.
+The short bar comes with **label** tabs but if you want to describe the tabs with an icon instead you have to set the label of each tab to ~No Symbol.
```
Comment (StefanIvanov) [inline en/components/tabs.md:39]:
```suggestion
The short bar comes with text content by default and if you want to show an icon instead, you have to assign a Material Icon to the `🔣 Icon` override and set the `🏷️ Label` to ~No Symbol for every tab in the bar.
```

File: en/components/tabs.md [hunk 3]
```diff
@@ -26,21 +26,27 @@ The Tabs can be **fixed** and can fill up the available horizontal space by adap
...
 
 <img class="responsive-img" src="../images/tabs_text.png" srcset="../images/tabs_text@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_icons.png" srcset="../images/tabs_icons@2x.png 2x" />
 
+## Tabs State
+
+The Tabs consists of tabs that support Active, Inactive and Disabled states. In Tabs there is always one Active tab and an arbitrary number of Inactive and Disabled ones.
```
Comment (StefanIvanov) [inline en/components/tabs.md:46]:
```suggestion
## Tab Item State

Tab items support Active, Inactive and Disabled states. In Tabs there is always one Active tab and an arbitrary number of Inactive and Disabled ones.
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/tabs.md [hunk 4]
```diff
@@ -26,21 +26,27 @@ The Tabs can be **fixed** and can fill up the available horizontal space by adap
 <img class="responsive-img" src="../images/tabs_fixed.png" srcset="../images/tabs_fixed@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_fluid.png" srcset="../images/tabs_fluid@2x.png 2x" />
 
-## Amount
+## Tabs Amount
 
-For the majority of scenarios, the Tabs need to contain between two and four tabs. If your case requires more than that, you may want to consider a fluid mode and represent only the tabs in view.
+For the majority of scenarios, the Tabs need to contain between two and four tabs. You can exclude a tab by setting it to ~No Symbol. If your case requires more tabs, you may want to consider a fluid mode and represent only the tabs in view.
```
Comment (StefanIvanov) [inline en/components/tabs.md]:
```suggestion
For the majority of scenarios, the Tabs need to contain between two and five tabs. You can remove a tab item by setting it to ~No Symbol. If your case requires more tabs, you may want to consider a fluid mode and represent only the tabs in view.
```

File: en/components/tabs.md [hunk 5]
```diff
@@ -26,21 +26,27 @@ The Tabs can be **fixed** and can fill up the available horizontal space by adap
...
 
 <img class="responsive-img" src="../images/tabs_2.png" srcset="../images/tabs_2@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_3.png" srcset="../images/tabs_3@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_4.png" srcset="../images/tabs_4@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/components/tabs.md:36]: add an image with five tabs

File: en/components/tabs.md [hunk 6]
```diff
@@ -26,21 +26,27 @@ The Tabs can be **fixed** and can fill up the available horizontal space by adap
 <img class="responsive-img" src="../images/tabs_fixed.png" srcset="../images/tabs_fixed@2x.png 2x" />
 <img class="responsive-img" src="../images/tabs_fluid.png" srcset="../images/tabs_fluid@2x.png 2x" />
```
Comment (StefanIvanov) [inline en/components/tabs.md:27]: the fluid image looks broken

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**